sklearn/utils/validation.py

```diff
@@ -2706,30 +2706,27 @@ def _to_object_array(sequence):
 
 
 def _check_feature_names(estimator, X, *, reset):
-    """Set or check the `feature_names_in_` attribute of an estimator.
+    """Check or set the `feature_names_in_` attribute for an estimator.
```

Comment:
I think `of` was perfectly fine. And why change the order of ``check`` and ``set`` ?
```suggestion
    """Set or check the `feature_names_in_` attribute of an estimator.
```